call-home/src/common/constants.rs

```diff
@@ -7,16 +7,10 @@ pub(crate) const PRODUCT: &str = "Mayastor";
 /// dir as an &str.
 const DEFAULT_ENCRYPTION_DIR_PATH: &str = "./";
 pub(crate) fn get_encryption_dir() -> String {
-    return match std::env::var("ENCRYPTION_DIR") {
-        Ok(input) => {
-            // This is a hack to eliminate a trailing slash.
-            match std::path::Path::new(&input).components().as_path().to_str() {
-                Some(path) => path.to_string(),
-                None => DEFAULT_ENCRYPTION_DIR_PATH.to_string(),
-            }
-        }
+    match std::env::var("ENCRYPTION_DIR") {
```

Comment:
nit: consider returning std::path::Path from these dir fn's.
nit: consider removing `get_` prefix from these functions

Reply:
I wanted to return Path, but it said `the trait bound `Path: std::marker::Sized` is not satisfied [E0277] `Path` does not have a constant size known at compile-time`

I'll remote the `get_` I guess.

Reply:
Ah right, it would need to be a `PathBuf`, nvm